api/envoy/extensions/filters/network/postgres_proxy/v3alpha/postgres_proxy.proto

```diff
@@ -29,4 +29,14 @@ message PostgresProxy {
   // are parsed. Parsing is required to produce Postgres proxy filter
   // metadata. Defaults to true.
   google.protobuf.BoolValue enable_sql_parsing = 2;
+
+  // Controls whether to terminate SSL session initiated by a client.
+  // If the value is false, the Postgres proxy filter will not try to
+  // terminate SSL session, but will pass all the packets to the upstream server.
+  // If the value is true, the Postgres proxy filter will try to terminate SSL
```

Comment:
If you have a link to the Postgres StartTls protocol that could be useful here.

Reply:
This one: https://www.postgresql.org/docs/current/protocol-flow.html#id-1.10.5.7.11

Reply:
Added link, but used _rel13_ instead of _latest_. As _latest_ moves forward the link may become invalid.

Reply:
It wont... the use of `current` keyword on docs URL is a common pattern in PostgreSQL community to lead you to the latest stable release. We do major releases once per year and what we do here with protocol decoding, etc don't change for last 17 years, so IMHO is better to use `current` instead of point to some specific version.

Reply:
OK. I will change to use `current` when I get more comments on PR.